Задача: Фильтры Instagram.

piligrim-curious opened the pull request:
**Image**: Класс изображений;
**FilterProcessor**: Класс обработки изображений.

piligrim-curious committed:
Задача: Фильтры Instagram.
Image: Класс изображений;
FilterProcessor: Класс обработки изображений.

SnurnicynKonstantin commented on src/main/java/school/faang/filters/instagram/FilterProcessor.java:
```diff
@@ -0,0 +1,47 @@
+    }
+
+    Function<Image, Image> combineFilters(Function<Image, Image> filterOne, Function<Image, Image> filterTwo) {
+        return image -> filterTwo.apply(filterOne.apply(image));
```
Можно ли заменить на `return filterOne.andThen(filterTwo);`?

piligrim-curious committed:
Задача: Фильтры Instagram.
Image: Класс изображений;
FilterProcessor: Класс обработки изображений.

Изменения:
(FilterProcessor.java)
* Изменён возврат функции.

SnurnicynKonstantin reviewed (approved):
Отлично, с функциями разобрался

SnurnicynKonstantin reviewed (approved):
Отлично, с функциями разобрался
